coding-concepts.js

```diff
@@ -7,53 +7,55 @@
 
 
 // --------------------1) What will this log?
 
 const cohort = "Alpha 2022"
-// console.log(cohort.length)
+ //console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This should come out with, the output of how many characters of text are in the string. This would be the "length" of the phrase in the sting it's output would be 10.
+// b) Verify and explain: I was correct, because it counts the amount of the legnth of the string. computers start counting at 0, so the 9 values in the string are counted as 10. 
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: this will run a greetin that says "Hello world!" 4 times in a row
+// b) Verify and explain: I was wrong, because it's asking for the 4th text value in the string. The output is o which is the 4th letter in the string.
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+ //console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: the output should be "Ruby"
+// b) Verify and explain: I was correct the output was Ruby, becuase the index spot is 1 and since computers count from zero. "Ruby" is in the 1 spot value of the index which gives you the awnser.
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
-
-// a) Your answer:
-// b) Verify and explain:
+ //console.log(weekendDays.toUpperCase())
 
+// a) Your answer: This will print the string array in all uppercase letters
+// b) Verify and explain: I got an error saying ".toUpperCase" is not a funtion. The reason it doesn't run because there is no value selected in the index.There seems to be errors in the code, ".toUpperCase" should be used it a code more like this. --->
+const sentence = 'The quick brown fox jumps over the lazy dog.';
+//console.log(sentence.toUpperCase());
+// the output should amount to; "THE QUICK BROWN FOX JUMPS OVER THE LAZY DOG."
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
-// console.log(typeof dataTypes.length)
+ //console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: It will log the length of the strings in the array. Either as a whole which would give a value of 18 or each string value which would be 6,6, and 7
+// b) Verify and explain: The output was number, i'm not sure why. I will need to look this up and brush up on this type of command.
```

Comment:
For sure!  This is another tricky one!  Length will give back a number that is the number of indexes.  There is another method inside the console log before getting the length to look at.  What does typeof do?

Reply:
working on the issues now and done